src/SigmaDSP.h

```diff
@@ -74,6 +74,7 @@ class SigmaDSP
 
     void volume_slew(uint16_t startMemoryAddress,   float dB, uint8_t slew = 12);
     void volume(uint16_t startMemoryAddress,        float dB);
+    void mute(uint16_t startMemoryAddress, bool flag);
```

Comment:
The formatting is a bit off here. Please align with the other arguments. And please use `state` instead of `flag`.